Makefile

```diff
@@ -45,6 +45,11 @@ buildx-push-image-deviceshifu: \
 	buildx-push-image-deviceshifu-http-socket \
 	buildx-push-image-deviceshifu-http-opcua
 
+buildx-push-image-telemetryService:
```

Comment:
let's be consistent and use `buildx-push-image-telemetry-service` instead.
Also the push statement should include all three platforms `--platform=linux/amd64,linux/arm64,linux/arm`